code_samples/api/graphql/src/GraphQL/Schema/MyCustomFieldDefinitionMapper.php

```diff
@@ -1,13 +1,14 @@
-<?php
+<?php declare(strict_types=1);
+
 namespace App\GraphQL\Schema;
 
-use Ibexa\GraphQL\Schema\Domain\Content\Mapper\FieldDefinition\DecoratingFieldDefinitionMapper
-use Ibexa\GraphQL\Schema\Domain\Content\Mapper\FieldDefinition\FieldDefinitionMapper;
+use Ibexa\Contracts\GraphQL\Schema\Domain\Content\Mapper\FieldDefinition\FieldDefinitionMapper;
+use Ibexa\GraphQL\Schema\Domain\Content\Mapper\FieldDefinition\DecoratingFieldDefinitionMapper;
```

Comment:
```suggestion
use Ibexa\Contracts\GraphQL\Schema\Domain\Content\Mapper\FieldDefinition\DecoratingFieldDefinitionMapper;
```
If you change one shouldn't the other follow as they have the same path?

Reply:
Nope, these are the specific namespaces, see: 
https://github.com/ibexa/graphql/blob/main/src/contracts/Schema/Domain/Content/Mapper/FieldDefinition/FieldDefinitionMapper.php#L7
https://github.com/ibexa/graphql/blob/main/src/lib/Schema/Domain/Content/Mapper/FieldDefinition/DecoratingFieldDefinitionMapper.php#L7